apcd-cms/src/apps/utils/table_filter.py

```diff
@@ -0,0 +1,8 @@
+import logging
```

Comment:
I think the purpose of the `utils/utils.py` file was for cases like this where the function didn't pertain to either the database or permissions. Would it make sense to add this table_filter to utils/utils.py that way this util folder doesn't bloat up eventually? Or is this to make it easier for future work and devs to figure out what's going on? If that's the case, this change can be ignored.